lib/src/file-input/types.ts

```diff
@@ -8,56 +7,56 @@ type Margin = {
 };
 type FileData = {
   error?: string;
-  file?: File;
+  file: File;
   preview?: string;
 };
 
 type Props = {
   /**
-   * Name attribute of the file input element.
+   * Name attribute.
    */
   name?: string;
   /**
-   * Text to be placed above the file input.
+   * Text to be placed above the component.
    */
   label?: string;
   /**
    * Uses one of the available file input modes:
    *    'file': Files are selected by clicking the button and selecting it through the file explorer.
    *    'filedrop': Files can be selected by clicking the button and selecting it through the file explorer or by dropping them inside the drag and drop area.
-   *    'dropzone':  Files can be selected by clicking the button and selecting it through the file explorer or by dropping them inside the drag and drop area. 
+   *    'dropzone':  Files can be selected by clicking the button and selecting it through the file explorer or by dropping them inside the drag and drop area.
    *     The drag and drop area of this mode is bigger than the one of the filedrop mode.
    */
-   mode?: "file" | "filedrop" | "dropzone";
+  mode?: "file" | "filedrop" | "dropzone";
   /**
-   * Helper text to be placed above the file input.
+   * Helper text to be placed above the component.
    */
   helperText?: string;
   /**
-   * Defines the file types accepted by the component. It is not possible to select a file with a non valid type.
+   * The file types that the component accepts. Its value must be one of all the possible values of the HTML file input's accept attribute.
    */
   accept?: string;
   /**
-   * An array of FileData representing the selected files. 
+   * An array of files representing the selected files.
    */
   value?: FileData[];
```

Comment:
Does an uncontrolled file input make sense? Maybe `callbackFile` or/and `value` should be mandatory, what do you think?

Reply:
Yes, file input is always controlled. I will change both to required!